Source/WebCore/css/html.css

```diff
@@ -242,7 +242,7 @@ table {
     display: table;
     border-collapse: separate;
     border-spacing: 2px;
-    border-color: gray;
+    border-color: currentColor;
```

Comment:
`currentColor` is the default value for `border-color`, so you can probably omit this line altogether.